protected/humhub/modules/comment/activities/NewComment.php

```diff
@@ -46,4 +52,16 @@ public function getDescription()
         return Yii::t('CommentModule.base', 'Whenever a new comment was written.');
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function getUrl()
+    {
+        $url = parent::getUrl();
```

Comment:
Wouldn't it be easier to just refer to the Perma Link Controller here?

Reply:
Done here https://github.com/humhub/humhub/pull/5168/commits/2f04aaf00a29d305222ffb5d7b14cc3cc8a52628#diff-e6cc3c96dbcaa1978cf2e96970550e14f594d514429b5e74c25022617824fe28R61.